iOS/Flows/Conversation/Cells/MessagesTimeMachineCollectionViewLayout.swift

```diff
@@ -100,11 +102,11 @@ class MessagesTimeMachineCollectionViewLayout: TimeMachineCollectionViewLayout {
                 let isScrolledToMostRecent
                 = (mostRecentOffset.y - collectionView.contentOffset.y) <= self.itemHeight
 
-                let isMostRecentInBottomSection = indexPath.item == self.numberOfItems(inSection: 1) - 1
+                let isNewMostRecent = indexPath.item == self.numberOfItems(inSection: 0) - 1
 
                 // Always scroll to the end for new user messages, or if we're currently scrolled to the
                 // most recent message.
-                if isMostRecentInBottomSection || isScrolledToMostRecent {
+                if isNewMostRecent && isScrolledToMostRecent {
```

Comment:
MAKE SURE TO CHECK OUT THE TIMEMACHINE LAYOUT CLASS! The diff is large so it may not show at first. :)